### ISSUE TITLE
[Accessibility- Screen Reader - Microsoft Form Pro - Email] Links does not have discernible text

### ISSUE THREAD

**Issue by sonimishra** [ckeditor/ckeditor4#3753]
## Type of report

Bug

User Experience:
User depending on screen reader might get affected if link does not have discernible text present on the screen and user will not be able to identify the control while navigating on screen.

## Provide detailed reproduction steps (if any)

Note: User credentials should NOT be included in the bug.

Repro Steps:

1. Open URL: https://forms.office-int.com/ and login using valid credentials.
2. Navigate to any created survey and select it.
3. Navigate Send button present at the top and select it.
4. Navigate to Email tab and select it. Email screen will appear.
5. Verify the issue.

### Expected result
 Links must have discernible text

### Actual result
Links does not have discernible text

MAS Reference:
[MAS 2.4.4 - Link Purpose (In Context)](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b06a891af-210c-4c6d-bb3e-92babab209e7%7d)
 
Reference Links
[Accessibility Insights!](https://apac01.safelinks.protection.outlook.com/?url=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx&data=02%7c01%7c%7cf03025479ea34f506ea808d68745348f%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636845128649252874&sdata=Ugu2ZTGi7gFcn4/v/i80oOUp2V6lESQ7IerJbSdYrpA%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
Please reach out to C&AI Teams channel for any process related queries.

* Browser: …
* OS: …1903 (OS Build 19018.1)
Tools: Accessibility insight tool
![MAS2 4 4_Link does not have text](https://user-images.githubusercontent.com/30456910/71657452-e8dd8580-2d65-11ea-9d89-a3f9c33f93f3.png)

* CKEditor version: …
* Installed CKEditor plugins: …

**Comment by jacekbogdanski** [comment]
Hello @sonimishra 

I'm a bit confused about how we could help because your reproduction steps seem to be related to the custom editor implementation. Could you provide reproduction steps including minimal CKEditor 4 demo using some online tool like https://codepen.io ?

Also note, that toolbar buttons are labeled by correct ARIA labels like `aria-labelledby`, `aria-describedby` with `button` role, so they shouldn't be treated as a links anymore. Not sure if your issue relates to toolbar buttons or content links. Could you refer to that also?

**Comment by jacekbogdanski** [comment]
Hi, It's been a while since we last heard from you. We're closing this issue for now. Still, feel free provide us requested feedback, so that we can reopen this issue.
